app/models/setting/provisioning.rb

```diff
@@ -72,7 +72,7 @@ def self.default_settings
       ),
       self.set(
         'excluded_facts',
-        N_("Exclude pattern for all types of imported facts (rhsm, puppet e.t.c.). Those facts won't be stored in foreman's database. You can use * wildcard to match names with indexes e.g. macvtap*"),
+        N_("Exclude pattern for all types of imported facts (puppet, ansible, rhsm). Those facts won't be stored in foreman's database. You can use * wildcard to match names with indexes e.g. ignore* will filter out ignore, ignore123 as well as a::ignore or even a::ignore123::b"),
```

Comment:
Plugins like Salt can add their own fact parser. I think they're also affected by this pattern so `etc` is ok to keep in I think.

Related: by reading this example it looks like the `networking::interfaces` issue I mentioned should already be covered.

Reply:
Hmm how about saying (puppet and plugins)?